### PR TITLE
TMDM-9923 : Ability to control the timeout or start/end date of the transaction in MDM components

### PR DESCRIPTION
**Pull request by qcui**

### CODE REVIEW

File: main/plugins/org.talend.designer.routines.mdmprovider/resources/java/routines/MDM.java [hunk 1]
```diff
@@ -437,6 +445,60 @@ public static String getLanguageVariant(String iso, String rawValue) {
...
+     *  
+     * {talendTypes} Long
+     * 
+     * {Category} MDM
+     * 
+     * {param} none
+     *  
+     * {example} getWSClientConnectionTimeout() # return 60000
+     */
+    public static Long getWSClientConnectionTimeout() throws Exception {
```
Comment (zfkang-talend) [inline main/plugins/org.talend.designer.routines.mdmprovider/resources/java/routines/MDM.java]: It's beeter to used int type.

File: main/plugins/org.talend.designer.routines.mdmprovider/resources/java/routines/MDM.java [hunk 2]
```diff
@@ -437,6 +445,60 @@ public static String getLanguageVariant(String iso, String rawValue) {
...
+     *  
+     * {talendTypes} Long
+     * 
+     * {Category} MDM
+     * 
+     * {param} none
+     *  
+     * {example} getWSClientReceiveTimeout() # return 60000
+     */
+    public static Long getWSClientReceiveTimeout() throws Exception {
```
Comment (zfkang-talend) [inline main/plugins/org.talend.designer.routines.mdmprovider/resources/java/routines/MDM.java]: It's beeter to used int type.

**Comment by zfkang-talend** [comment]
Validated.

**Comment by pbabin** [comment]
It would have been great to not duplicate code (exactly same logic is used to extract the value of the 2 JVM parameters)

**Pull request merged**